test/unit-test/README.md

```diff
@@ -58,48 +59,93 @@ BUILD_DIR="${UNIT_TEST_DIR}/build/"
+FreeRTOS_DNS.c                      | 100%   361| 100%  23| 100%  184
+FreeRTOS_DNS_Cache.c                | 100%   126| 100%  11| 100%   54
+FreeRTOS_DNS_Callback.c             | 100%    80| 100%   4| 100%   34
+FreeRTOS_DNS_Networking.c           | 100%    31| 100%   5| 100%    4
+FreeRTOS_DNS_Parser.c               | 100%   366| 100%   6| 100%  192
+FreeRTOS_Stream_Buffer.c            | 100%   101| 100%  12| 100%   46
+FreeRTOS_Tiny_TCP.c                 | 100%    92| 100%  12| 100%   40
+
+[/home/ubuntu/FreeRTOS-Plus-TCP/test/unit-test/build/Annexed_TCP_Sources/]
+FreeRTOS_BitConfig.c                | 100%    87| 100%  11| 100%   30
+FreeRTOS_DHCP.c                     | 100%   444| 100%  19| 100%  256
+FreeRTOS_DHCPv6.c                   | 100%   576| 100%  18| 100%  252
+FreeRTOS_ICMP.c                     | 100%    44| 100%   3| 100%   11
+FreeRTOS_IP.c                       | 100%   642| 100%  46| 100%  331
+FreeRTOS_IP_Timers.c                | 100%   152| 100%  21| 100%   84
+FreeRTOS_IP_Utils.c                 | 100%   472| 100%  33| 100%  256
+FreeRTOS_IPv4.c                     | 100%   148| 100%   7| 100%  110
+FreeRTOS_IPv4_Sockets.c             | 100%    62| 100%   4| 100%   40
+FreeRTOS_IPv4_Utils.c               | 100%    40| 100%   2| 100%   16
+FreeRTOS_IPv6.c                     | 100%   201| 100%  10| 100%  126
+FreeRTOS_IPv6_Sockets.c             | 100%   189| 100%  12| 100%  106
+FreeRTOS_IPv6_Utils.c               | 100%    99| 100%   5| 100%   49
+FreeRTOS_ND.c                       | 100%   431| 100%  18| 100%  187
+FreeRTOS_RA.c                       | 100%   198| 100%   9| 100%   80
+FreeRTOS_Routing.c                  | 100%   360| 100%  24| 100%  213
+FreeRTOS_Sockets.c                  | 100%  1542| 100%  89| 100%  841
+FreeRTOS_TCP_IP.c                   | 100%   243| 100%  10| 100%  196
+FreeRTOS_TCP_Reception.c            | 100%   157| 100%   5| 100%   87
+FreeRTOS_TCP_State_Handling.c       | 100%   277| 100%   9| 100%  124
+FreeRTOS_TCP_State_Handling_IPv4.c  | 100%    52| 100%   1| 100%   30
+FreeRTOS_TCP_State_Handling_IPv6.c  | 100%    50| 100%   1| 100%   32
+FreeRTOS_TCP_Transmission.c         | 100%   331| 100%  19| 100%  154
+FreeRTOS_TCP_Transmission_IPv4.c    | 100%   131| 100%   3| 100%   36
+FreeRTOS_TCP_Transmission_IPv6.c    | 100%   145| 100%   3| 100%   50
+FreeRTOS_TCP_Utils.c                | 100%    21| 100%   2| 100%   16
+FreeRTOS_TCP_Utils_IPv4.c           | 100%     9| 100%   1| 100%    4
+FreeRTOS_TCP_Utils_IPv6.c           | 100%    15| 100%   1| 100%    6
+FreeRTOS_TCP_WIN.c                  | 100%   456| 100%  37| 100%  222
+FreeRTOS_UDP_IP.c                   | 100%    28| 100%   2| 100%   12
+FreeRTOS_UDP_IPv4.c                 | 100%   128| 100%   2| 100%   84
+FreeRTOS_UDP_IPv6.c                 | 100%   143| 100%   4| 100%   84
+=====================================================================
+                              Total:| 100%  9388| 100% 525| 100% 4920
```

Comment:
Did line coverage really increase that much? If so why?

Reply:
The line, branch coverages are 100% from [V3.0.0](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/releases/tag/V3.0.0) released on Aug 10, 2022.
It's just that the sample output examples for the unit test readme scripts were not updated, as those are just for reference.